Conservation.pm

```diff
@@ -189,35 +201,65 @@ sub run {
   my $chr = $vf->{chr};
   $chr =~ s/^chr//i;
 
-  $parser->seek($chr, $vf->{start} - 1, $vf->{end});
-  $parser->next;  
-  return $parser->get_raw_score ? { Conservation => sprintf("%.3f", $parser->get_raw_score)} : {};
+  #Check if insertion and adjust to capture flanking bases
+  if ($vf->{start} - 1 == $vf->{end}){
+    $parser->seek($chr, $vf->{start} - 2, $vf->{end} + 1);
+  }
+  else{
+    $parser->seek($chr, $vf->{start} - 1, $vf->{end});
+  }
+  # Grab the score
+  my @values = ();
+  push @values, $parser->{waiting_block}[3];
```

Comment:
`$parser->{waiting_block})` can also have feature interval > 2, do we need to capture multiple scores in the interval?
Example for testing: `17 64015480 64015484 GGCTA G`